lifecycle.go

```diff
@@ -6,18 +6,28 @@ import (
 	"time"
 
 	"github.com/d3sw/floop/handlers"
+	"github.com/d3sw/floop/resolver"
 	"github.com/d3sw/floop/types"
 )
 
+var (
+	resolverPort = 8600
+	resolverHost = "127.0.0.1"
```

Comment:
We might want to make this configurable. floop as a process will be running on our nomad servers. The resolvers we would be querying exist on consul, which would be completely different servers.

I think the default host should be `consul.service`, but we should add some configuration option to set both the resolver host and port should we want to override.